Zend/tests/bug79514.phpt

```diff
@@ -2,12 +2,15 @@
 Bug #79514 (Memory leaks while including unexistent file)
 --FILE--
 <?php
-$mem1 = memory_get_usage(true);
-for ($i = 0; $i < 100000; $i++) {
-    @include __DIR__ . '/bug79514.doesnotexist';
+$filename = __DIR__ . '/bug79514.doesnotexist';
+// approximate size of the zend_string
+$size = 8 + 3 * PHP_INT_SIZE + strlen($filename) & ~(PHP_INT_SIZE - 1);
+$mem1 = memory_get_usage();
+for ($i = 0; $i < 100; $i++) {
+    @include $filename;
```

Comment:
anyway, can php engine be updated to not leak also when same existing file is included multiple times?

Reply:
Yes, see below.